Sources/Profile-Modify.php

```diff
@@ -500,7 +500,7 @@ function $preload:		A function that is used to load data required for this eleme
 						log_error(sprintf($txt['smiley_set_dir_not_found'], $set_names[array_search($set, $context['smiley_sets'])]));
 
 						$context['smiley_sets'] = array_filter($context['smiley_sets'], function($v) use ($set)
-							{
+						{
```

Comment:
If the indentation is reduced on this line, it should be reduced on the following lines as well.